examples/ts-without-babel/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "target": "es2018",
```

Comment:
We probably want this to be `es5`/`es6` instead for compatibility?

Reply:
Changed to es6, but I'm not sure. We use `"module": "esNext"` (dynamic import) anyway and es6 is outdated. Maybe it's okay for example project.

Reply:
For full IE11 compatibility we will need ES5 (or ES6 if we're aiming for major browser compatibility, I think this is a good thing to do), then it's about the module resolution target - ESNext (I've heard CommonJS + `moduleResolution: node` works, but I'm not sure). This is a bit complicated because we're using Webpack, and Webpack handles the dynamic import syntax for us instead of letting it bleed to userland - so module `esnext` works sorta independently with the target ...